openlane/common/toolbox.py

```diff
@@ -416,10 +434,9 @@ class State(IntEnum):
             return out_path
 
         commands = ""
-        for define in list(defines) + ["NO_PRIMITIVES"]:
+        for define in list(defines):
             commands += f"verilog_defines -D{define};\n"
-        for file in final_files:
-            commands += f"read_verilog -sv -lib {file};\n"
+        commands += f"read_verilog -pwires -sv -lib {out_path};\n"
```

Comment:
what does `-pwries` do?

Reply:
Turned parameters into wires, but we're no longer handling those so